WORKSPACE

```diff
@@ -88,3 +88,32 @@ pip_install(
     name = "grpc_python_dependencies",
     requirements = "@com_github_grpc_grpc//:requirements.bazel.txt",
 )
+
+http_archive(
+    name = "build_bazel_rules_swift",
```

Comment:
comments on why this part is needed ?

Reply:
It’s needed by PodToBUILD which converts Cronet pod to bazel build.
https://bazel.build/migrate/cocoapods

Reply:
sg. https://github.com/grpc/grpc-ios/issues/104 will track the task to introduce bazel based CronetFramework build so we can remove this cocoapod dependency in our bazel tool tree.